testdata/p4_16_samples_outputs/uninit-frontend.p4

```diff
@@ -33,12 +31,9 @@ parser p1(packet_in p, out Header h) {
         }
     }
     state next1 {
-        d_0 = false;
         transition next3;
     }
     state next2 {
-        c_0 = true;
-        d_0 = c_0;
```

Comment:
please check also that these assignments are truly dead.

Reply:
As far as I can tell (mostly by looking at the output), they do seem truly dead.